public_html/wp-content/plugins/wcpt/wcpt-meetup/class-meetup-admin.php

```diff
@@ -478,6 +480,58 @@ public function maybe_update_organizers( $new_status, $old_status, $post ) {
 
 		}
 
+		/**
+		 * Send notification to slack when a Meetup becomes active in the chapter or is declined.
+		 *
+		 * @param string  $new_status
+		 * @param string  $old_status
+		 * @param WP_Post $meetup
+		 *
+		 * @return null|bool Will be null if notification was not enabled, or false if notifcation was attempted but failed. true if notification was successful
+		 */
+		public function notify_application_status_in_slack( $new_status, $old_status, WP_Post $meetup ) {
+
+			$notification_enabled = apply_filters( 'meetup_application_notification_enabled', true );
+
+			if ( ! $notification_enabled ) {
+				return null;
+			}
+
+			if ( 'wcpt-mtp-active' === $new_status ) {
+				return $this->notify_new_meetup_group_in_slack( $meetup );
```

Comment:
so both of these approaches works. but I agree that `self::` feels more correct. will change that.
its static so that we can easily call these from cli if need be.